templates/bastion.cfn.yml

```diff
@@ -39,33 +42,35 @@ Mappings:
   # Note: This has not been tested with Amazon Linux 2
   AMIMap:
```

Comment:
The change rate of this mapping is going to be high. One thought that occurs to me is moving to an `AWS::Include` transform that holds this mapping. You could then programmatically generate the latest AMI using a script such as https://gist.github.com/jpignata/990c9bd3a7d6a4fd93b1be20bd7a0776.

This is on my list for a few templates I manage. Probably not for this PR as it's large enough already.

Reply:
Agreed. Created an issue in the backlog.